test/cypress/integration/04-organization_v2.test.js

```diff
@@ -3,17 +3,35 @@ describe('New organization profile', () => {
    * Contributions section is already tested in `05-user_v2.test.js`
    * About section is already tested in `04-collective_v2.test.js`
    */
-
+  before(() => {
+    cy.createCollective({ type: 'ORGANIZATION' }).then(collective => {
+      const collectiveSlug = collective.slug;
+      cy.visit(`/${collectiveSlug}/`);
+    });
+  });
   describe('Contributors section', () => {
-    it.skip('Only shows core contributors without any filter', () => {
-      // TODO
+    it('Only shows core contributors without any filter', () => {
+      cy.getByDataCy('filters').should('not.exist');
+      cy.getByDataCy('section-contributors').click({ force: true });
+      cy.hash().should('eq', '#section-contributors');
+
+      cy.getByDataCy('section-contributors-title').contains('Core contributors');
+      cy.getByDataCy('ContributorsGrid_ContributorCard');
```

Comment:
Not necessary (we already check that below)

```suggestion
```

Reply:
Let me remove it ASAP.